src/tests/ftest/io/posix_stat.yaml

```
@@ -0,0 +1,22 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+  test_clients:
+    - client-A
+timeout: 600
+pool:
+  mode: 511
+  name: daos_server
+  scm_size: 1073741824
```

We don't need `mode:` anymore and we should use human-readable sizes:
```suggestion
  name: daos_server
  scm_size: 1G
```